applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -16,6 +16,19 @@
 
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
 
+#define LE_AUDIO_EXTENDED_ADV_NAME                                                                 \
+	BT_LE_ADV_PARAM(BT_LE_ADV_OPT_EXT_ADV | BT_LE_ADV_OPT_USE_NAME,                            \
+			CONFIG_BLE_ACL_EXT_ADV_INT_MIN, CONFIG_BLE_ACL_EXT_ADV_INT_MAX, NULL)
+
+#define LE_AUDIO_EXTENDED_ADV_CONN_NAME                                                            \
```

Comment:
Why is "NAME" used?

Reply:
Because of the option: BT_LE_ADV_OPT_USE_NAME. That is also why this one includes CONN, as the option BT_LE_ADV_OPT_CONNECTABLE is set.